libraries/plugins/xfpga/usrclk/fpga_user_clk.c

```diff
@@ -501,15 +522,20 @@ fpga_result usrclk_calibrate(uint8_t *uio_ptr, uint8_t *seq)
 
 fpga_result get_usrclk_uio(const char *sysfs_path,
 	uint32_t feature_id,
+	const char* guid,
```

Comment:
define enum_fpga_feature() or find_fpga_feature() new function in common.c file, it enums fpga features and returns dfl_dev.x